app/javascript/packs/pages/home/VolunteerCallout.jsx

```diff
@@ -39,19 +39,16 @@ const VolunteerCallout = () => (
           {/* Intentionally empty to create an even number of columns */}
         </Grid> 
         <Grid item xs={4} sm={2}>
```

Comment:
Since we dropped the Instagram link, we can tidy up the columns here. Maybe 4x1 on desktop and 2x2 on mobile? what do you think?

Reply:
Yeah nice catch :+1: . I'll look into it

Reply:
Here's how the icons would look like in mobile view:

![Screenshot from 2020-03-27 23-51-41](https://user-images.githubusercontent.com/15841903/77814294-1ea0ba80-7086-11ea-8535-7de01f95ed85.png)